intelligence-services/latest/using/index.md

```diff
@@ -142,3 +142,21 @@ This example shows you how to view these properties in Digital Workspace.
 7. To search by a custom property, type `schema:<category>:<content>` in the search field and press **Enter**.
 
     For example, enter `schema:sport:runner`
+
+## View Transcription
+
+When configured, transcripts of your audio and video files are generated automatically within the Digital Workspace, including indexing and metadata generation which allows you to search their content easier. Captions of the transcripts can be automatically placed on top of the audio and video content, see the images below. For information on how to configure this in the Digital Workspace see [Set up a folder rule]({% link intelligence-services/latest/using/index.md %}#set-up-a-folder-rule).
+
+**Transcription**
+![transcript]({% link intelligence-services/images/text-transcript.png %})
```

Comment:
As you mentioned, images approved by PM.

For consistency, it would be great to add a view section for all other AI renditions as well, ie.

- detecting objects (labels) within an image (via "aiLabels")
- detecting text (~ OCR) within an image (using "aiTextract")
- detecting features (aka named entity recognition) within text (using "aiFeatures")

Reply:
Hi @montgolfiere thanks for this review. 
The only part I've done for this PR and docs JIRA is the changing of the default renditions Note:
Are you happy for it to now be in that format? I'm in two minds actually looking at it again. 
All the other bits you mention I shall add to DOCS-6242 and make a new PR for it. Will discuss those requirements in more detail once this PR and current DOCS ticket are done.
Thanks.